src/components/Person/Person.jsx

```diff
@@ -1 +1,25 @@
-// export const Person = ({ person }) => ();
+import React from 'react';
+import './Person.scss';
+
+export const Person = ({ person }) => {
+  const { name, age, sex, isMarried, partnerName } = person;
+
+  const personPartner = sex === 'm'
+    ? 'wife'
+    : 'husband';
+  const personMaried = isMarried
+    ? `${partnerName} is my ${personPartner}`
+    : 'I am not married';
+
+  return (
+    <section className="Person">
+      <h2 className="Person__name">{`My name is ${name}`}</h2>
```

Comment:
hard to read, why write everything in one line?
```suggestion
      <h2 className="Person__name">
        {`My name is ${name}`}
      </h2>
```